Sources/Yams/Emitter.swift

```diff
@@ -183,9 +183,31 @@ public final class Emitter {
 
     public var data = Data()
 
-    let documentStartImplicit: Int32
-    let documentEndImplicit: Int32
-    let version: (major: Int, minor: Int)?
+    public struct Options {
+        /// Set if the output should be in the "canonical" format as in the YAML specification.
+        public var canonical: Bool = false
+        /// Set the intendation increment.
+        public var indent: Int = 0
+        /// Set the preferred line width. -1 means unlimited.
+        public var width: Int = 0
+        /// Set if unescaped non-ASCII characters are allowed.
+        public var allowUnicode: Bool = false
+        /// Set the preferred line break.
+        public var lineBreak: LineBreak = .ln
+
+        // internal since we don't know these should be exposed.
```

Comment:
missing a word here: "internal since we don't know if these should be exposed." or maybe "how these should be exposed." depending on what you mean.

Reply:
Thanks!
I don't mean *how*, maybe *if*.